core/src/scss/components/button/_button--secondary.scss

```diff
@@ -1,7 +1,22 @@
 @charset "UTF-8";
 
 // Secondary button.
-.su-button--secondary,
-.su-button--secondary.su-link {
-  @include button-secondary;
+.su-button {
+  &--secondary {
+    &,
+    &.su-link {
+      @include button-secondary;
+    }
+  }
+}
+
+button {
+  &.su-button {
+    &--secondary {
+      &,
+      &.su-link {
+        @include button-secondary;
```

Comment:
Same comment as above - I believe we have `<a>` with the `.su-button` and `.su-link` classes, but I don't think we have `<button>` with `.su-link` class.